.github/workflows/test-ci.yml

```diff
@@ -0,0 +1,43 @@
+name: Test CI
+
+on:
+  # Triggers the workflow on push and pull request events but only for pull_requests on the main branch
+  push:
+    branches:
+      - main
+
+  pull_request:
+    branches:
+      - main
+
+  # Allows you to run this workflow manually
+  workflow_dispatch:
+
+jobs:
+  build:
```

Comment:
Could you split mypy and pytest into separate jobs? It'll allow them to run in parallel and we can spot at a glance if one of them failed without having to click on them

Reply:
That's an implementation error on my part. It should be fixed with the asyncio browsers PR.